### DISCUSSION TITLE
Seting $(TargetName) doesn't change the name of the generated .pri file

### DISCUSSION THREAD

**Discussion by cosmin42** [microsoft/WindowsAppSDK#2328] [Dev team discussions]
Building a WinUI3 application which was just created, like the one attached, a `resources.pri` file is generated.

The name of the .pri file should be $(RootNamespace).pri which in this case is ExampleApp.pri.

Under the default generated vcxproj there is the below comment

```
    <!--
      $(TargetName) should be same as $(RootNamespace) so that the produced binaries (.exe/.pri/etc.)
      have a name that matches the .winmd
    -->
```

Setting the Target name as $(RootNamespace) doesn't change the name of resources.pri .

This creates a further problem when another WinUI3 project depends of the fist one because the .pri files will have the same name and there will be a merge problem like in the error below:

```
20>   Creating library C:\Users\username\Windows\src\\generated\output\x64\Debug\ProjectB\ProjectB.lib and object C:\Users\username\Windows\src\\generated\output\x64\Debug\ProjectB\ProjectB.exp
20>ProjectB.vcxproj -> C:\Users\username\Windows\src\generated\output\x64\Debug\ProjectB\ProjectB.dll
20>C:\Users\username\Windows\src\packages\Microsoft.WindowsAppSDK.1.0.1\build\Microsoft.Build.Msix.Packaging.targets(1379,5): error APPX1101: Payload contains two or more files with the same destination path 'resources.pri'. Source files: 
20>C:\Users\username\Windows\src\packages\Microsoft.WindowsAppSDK.1.0.1\build\Microsoft.Build.Msix.Packaging.targets(1379,5): error APPX1101: C:\Users\username\Windows\src\generated\output\x64\Debug\ProjectB\resources.pri
20>C:\Users\username\Windows\src\packages\Microsoft.WindowsAppSDK.1.0.1\build\Microsoft.Build.Msix.Packaging.targets(1379,5): error APPX1101: C:\Users\username\Windows\src\generated\output\x64\Debug\ProjectA\resources.pri
20>Done building project "ProjectB.vcxproj" -- FAILED.
```

This is the case for both the 1.0.1 and 1.0.0 versions of WindowsAppSDK.

I attached a simple WinUI3 project that generates resources.pri instead of ExampleApp.pri .

[ExampleApp.zip](https://github.com/microsoft/WindowsAppSDK/files/8398417/ExampleApp.zip)

**Comment by marb2000** [comment]
Thanks, @cosmin42 for the question. 

**Resources.pri is the final name** for the app’s .pri file. The comment in the project template concerns the naming of the *intermediate* .pri file that gets produced as part of the build. So the behavior they’re seeing **is by design**.

Hope this helps to clarify your doubt.

**Comment by cosmin42** [comment]
@marb2000  thanks for answering, if the `resources.pri` is the right naming then what could be the reason of the error "Payload contains two or more files with the same destination path"? Is there a configuration that I can do to change the name of `resources.pri`?
I attached a Diagnostic build log.

I tried different versions of WinAppSdk (1.0.0, 1.02, the preview version) but I get the same error.
[build_log.txt](https://github.com/microsoft/WindowsAppSDK/files/8521444/build_log.txt)

**Comment by weitzhandler** [comment]
Also struggling with this issue.
Spent an hour and cannot find a solution.
